test/math.jl

```diff
@@ -72,22 +72,22 @@ end
         @test y == 0
 
         # more ldexp tests
-        @test ldexp(T(0.8), 4) === T(12.8)
-        @test ldexp(T(-0.854375), 5) === T(-27.34)
+        @test ldexp(T(0.8), 4) == T(12.8)
```

Comment:
Why test less specifically? `===` verifies that the types are the same. Would it fail here?